.circleci/config.yml

```diff
@@ -237,6 +246,10 @@ commands:
                 command: |
                   brew install curl
                   echo 'export PATH="/usr/local/opt/curl/bin:$PATH"' >> $BASH_ENV
+            - run:
+                name: Install cmake
+                command: |
+                  brew install cmake
```

Comment:
It's unlikely that you'll get versioning issues with cmake, but consider:

```
brew install cmake@3.27.2
```
or, possibly better, don't use brew at all and just do the same thing you do on the linux install.

Reply:
113ed44

Reply:
https://app.circleci.com/pipelines/github/apollographql/federation-rs/2408/workflows/64615fc5-a0f7-40a5-b99b-17cdcf5da796/jobs/10928 ok i've tried hard enough, i will not pin to a specific version of cmake just yet. I'll add a todo.

Reply:
todo added in [8b88782](https://github.com/apollographql/federation-rs/pull/398/commits/8b887820523a2a65e2add4f268b4d2ca0ca7595d)